### PR TITLE
Script for updating timestamp format in `.dandi/assets.json`

### PR DESCRIPTION
**Pull request by jwodder**
For https://github.com/dandi/backups2datalad/issues/40

### CODE REVIEW

File: tools/update-timestamp-format.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                        (
+                            "Update asset timestamp properties format in"
+                            " .dandi/assets.json"
+                        ),
+                        ".dandi/assets.json",
+                    ],
+                    cwd=p,
+                    check=True,
+                )
+                to_save.append(p.name)
```
Comment (yarikoptic) [inline tools/update-timestamp-format.py:53]: in principle you could avoid "manual" `git commit ` above  and just add `fpath` to `to_save` and then rely on `datalad save` to do the right thing to save all those individual files (and their subdatasets).

**Review by yarikoptic — COMMENTED** [review]
committing/saving could be made into a single `datalad save` command but otherwise -- should work.

**Review by yarikoptic — APPROVED** [review]
intended to approve, not comment ;-)

File: tools/update-timestamp-format.py (hunk 1 above)
Comment (jwodder) [reply tools/update-timestamp-format.py:53]: Too late for that now.

**Pull request merged**